garage/envs/mujoco/maze/maze_env.py

```diff
@@ -266,6 +265,7 @@ def _find_robot(self):
                 if structure[i][j] == 'r':
                     return j * size_scaling, i * size_scaling
         assert False
```

Comment:
this will always crash.

Reply:
Remove `assert False` and add `return None, None` instead?

Reply:
oh i see. it is a way of throwing an error if nothing was found.

replace it with an exception and we're good.

Reply:
👍